package.json

```diff
@@ -48,9 +48,12 @@
     "nyc": "^15.1.0",
     "sinon": "^9.2.1",
     "ts-mocha": "^7.0.0",
-    "typescript": "^4.0.3"
+    "typescript": "^4.9.5"
   },
   "dependencies": {
-    "discord.js": "^13.0.1"
+    "discord.js": "^14.7.1"
+  },
+  "overrides": {
```

Comment:
Why is this required?

Reply:
There was an issue with the discord-api-types for v14, it was something mentioned in the Discord.JS discord FAQ channel but it's possible that they fixed it with the 14.8.0 release. I'll update the package and see whether I get the problem or not (was an issue with the guild channel types not being correct for sending a message in them)